openedx/features/enterprise_support/enrollments/utils.py

```diff
@@ -18,14 +22,17 @@
 log = logging.getLogger(__name__)
 
 
-def lms_enroll_user_in_course(
+def lms_update_or_create_enrollment(
     username,
     course_id,
-    mode,
-    enterprise_uuid,
+    desired_mode,
+    enterprise_uuid=None,
     is_active=True,
```

Comment:
Should this be changed to is_active=None so as not to cause callers to inadvertently activate an inactive enrollment?  It's pretty corner-case-ey, but something I can imagine still being a future head scratcher.

Reply:
I guess it would only default to True in the create branch

Reply:
That's a good point, I think I'll force it to be required